### PR TITLE
[doc] Add an example.dart file

### PR DESCRIPTION
**Pull request by ayan-b**
Pana suggests to add an example.dart file
here: https://pub.dev/packages/yaml#-analysis-tab-

Closes https://github.com/dart-lang/yaml/issues/45

### CODE REVIEW

File: example/example.dart [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import 'package:yaml/yaml.dart';
+
+// ignore: always_declare_return_types
```
Comment (kevmoo) [inline example/example.dart]: Instead of having the ignore in the example, just add a return type! `void~

File: example/example.dart [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import 'package:yaml/yaml.dart';
```
Comment (kevmoo) [inline example/example.dart]: Add standard code copyright here – with year 2020 – see other code files

File: example/example.dart (hunk 1 above)
Comment (ayan-b) [reply example/example.dart]: Done!

File: example/example.dart (hunk 2 above)
Comment (ayan-b) [reply example/example.dart]: Done!

**Pull request merged**